Documentation/connectors-configuration.md

```diff
@@ -52,6 +52,8 @@ This connector config lets users authenticate with other OIDC providers. In addi
 
 * trustedEmailProvider: a `boolean`. If true dex will trust the email address claims from this provider and not require that users verify their emails.
 
+* domain: a `string` (optional). If set to a non-empty string, users that are unable to log in to the remote Identity Provider with an email address whose domain matches the provided value will be prevented from logging in to or registering with dex. For example, if `domain` is set to `example.com`, user `foo@example.com` will be granted an identity token from dex, but `bar@example.net` will not.
```

Comment:
* domain: a `string` (optional). If set, users' email address domain must match the provided value. Email addresses that do not match will not be allowed to register or log in with dex. For example, if `domain` is set to `example.com`, the email address `user@example.com` will be granted an identity token, but `user@example.net` will not.

(an attempt to clarify a graf that confused me)